### DISCUSSION TITLE
phase_clouser_bias

### DISCUSSION THREAD

**Discussion by sisi-ali** [insarlab/MintPy#837] [Q&A]
hi 
I work with MintPy software and I want to get phase_clouser_bias, but it is not possible, I use this code (https://github.com/insarlab/MintPy/blob/main/mintpy/closure_phase_bias.py)
Thank you for guiding me

**Comment by yunjunz** [comment]
Please provide enough detailed descriptions, for more informed dialogue.

**Comment by sisi-ali** [comment]
Hi dear,
I would like  to get the phase bias effect using software MintPy. 
For this, from the closure_phase_bias.py library. but I get an error and I can't do it
Example of closure phase calculated from an 18-day interferogram  and three 6-day interferograms

**Comment by sisi-ali** [comment]
Hi dear,
I would like to get the phase bias effect using software MintPy.
For this, from the closure_phase_bias.py library. but I get an error and I can't do it
Example of closure phase calculated from an 18-day interferogram and three 6-day interferograms
